mmcv/runner/base_runner.py

```diff
@@ -353,6 +354,9 @@ def resume(self,
                 self.logger.info('the iteration number is changed due to '
                                  'change of GPU number')
 
+        if resume_meta:
```

Comment:
How about we make it a default behavior to load checkpoint['meta'] if the key 'meta' exists?

Reply:
I agree.